### PR TITLE
[test-only] Returning space link test

### PR DESCRIPTION
**Pull request by ScharfViktor**
- [x] Returning space link
- [x] added internal link to space test

### CODE REVIEW

File: tests/e2e/cucumber/features/smoke/spaces/publicLink.ocis.feature [hunk 1]
```diff
@@ -14,31 +14,37 @@ Feature: spaces public link
...
       | name | id     |
       | team | team.1 |
+    And "Alice" creates the following folder in space "team" using API
+      | name        |
+      | spaceFolder |
+    And "Alice" adds the following members to the space "team" using API
+      | user  | role       | shareType |
+      | Brian | Can edit   | space |
+      | Carol | Can view   | space |
+      | David | Can manage | space |
```
Comment (SwikritiT) [inline tests/e2e/cucumber/features/smoke/spaces/publicLink.ocis.feature:24]: The indent is not right here
```suggestion
      | user  | role       | shareType |
      | Brian | Can edit   | space |
      | Carol | Can view   | space |
      | David | Can manage | space |
```

File: tests/e2e/cucumber/features/smoke/spaces/publicLink.ocis.feature [hunk 2]
```diff
@@ -14,31 +14,37 @@ Feature: spaces public link
...
+
```
Comment (SwikritiT) [inline tests/e2e/cucumber/features/smoke/spaces/publicLink.ocis.feature:25]:
```suggestion
```

File: tests/e2e/cucumber/features/smoke/spaces/internalLink.ocis.feature [hunk 3]
```diff
@@ -18,22 +19,31 @@ Feature: internal link share in project space
     And "Alice" creates the following file in space "Marketing" using API
       | name              | content     |
       | myfolder/plan.txt | secret plan |
+    And "Alice" adds the following members to the space "Marketing" using API
+      | user  | role       | shareType |
+      | Brian | Can edit   | space |
+      | Carol | Can view   | space |
```
Comment (saw-jan) [inline tests/e2e/cucumber/features/smoke/spaces/internalLink.ocis.feature:25]:
```suggestion
    And "Alice" adds the following members to the space "Marketing" using API
      | user  | role       | shareType |
      | Brian | Can edit   | space     |
      | Carol | Can view   | space     |
```

**Review by saw-jan — COMMENTED** [review]
Others LGTM :+1:

**Review by saw-jan — APPROVED** [review]

**Pull request merged**